### PR TITLE
IRIS-4246 | post-CR fixes

### PR DESCRIPTION
**Pull request by tortila**
Original PR: https://github.com/Wikia/selenium-tests/pull/1923
Tests are still passing: http://jenkins.wikia-prod:8080/view/Iris/view/sandbox/job/iris-auth-forgotten-password-sandbox/20/

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserPool.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.wikia.webdriver.common.core.helpers;
+
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+
+public class ForgottenPasswordUserPool {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserPool.java]: MAJOR Add a private constructor to hide the implicit public one. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)

File: src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserPool.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  private static final Credentials CREDENTIALS = Configuration.getCredentials();
+
+  public static UserWithEmail user1 = new UserWithEmail(User.FORGOTTEN_PASSWORD,
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserPool.java]: MAJOR Make user1 a static final constant or non-public and provide accessors if needed. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
MAJOR Make this "public static user1" field final [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1444)

File: src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserPool.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    CREDENTIALS.forgottenPasswordEmail1Address,
+    CREDENTIALS.forgottenPasswordEmail1Password);
+
+  public static UserWithEmail user2 = new UserWithEmail(User.FORGOTTEN_PASSWORD_SPACES,
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserPool.java]: MAJOR Make user2 a static final constant or non-public and provide accessors if needed. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
MAJOR Make this "public static user2" field final [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1444)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 5 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserPool.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+  private static final Credentials CREDENTIALS = Configuration.getCredentials();
+
+  @Getter
+  private UserWithEmail user1 = new UserWithEmail(User.FORGOTTEN_PASSWORD,
+    CREDENTIALS.forgottenPasswordEmail1Address,
+    CREDENTIALS.forgottenPasswordEmail1Password);
+
+  @Getter
+  private UserWithEmail user2 = new UserWithEmail(User.FORGOTTEN_PASSWORD_SPACES,
```
Comment (slayful) [inline src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserPool.java]: `userWithSpaces` rather than `user2`?

**Pull request merged**